### DISCUSSION TITLE
Suggesting for improving Topic managment

### DISCUSSION THREAD

**Discussion by mbaysek** [jendrikseipp/rednotebook#735] [Ideas]
Right now RN is an amazing tool.  The tags let you categorize all the stuff into easily locatable resources.  I use my journal as a reference a lot.

But one thing it's missing, is something that another software has, and that's topic management.  E-Sword bible software lets you track notes by date in the Journal feature, and also by verse and chapter very easily.  I look at this Journal Feature as similar to the calendar journal in RN.  It's very easy to take notes that happen on a particular day and tag them with the useful identifiers, even topical identifiers that anyone might want to use while using the software.  

But all of the notes are in different files, based on the date they were written.   But E-Sword also has Study Notes which are based on chapter and verse, Topic notes which is a flat namespace of topics, and also study notes.  E-Sword is open source free software and even if you don't study bible, you can appreciate the way the software is organized.

It would be nice if we could add a Topic and keep a file for each Topic.   Taking this a step further, we could consider a topic a "folder" and allow people to create folders that contain files, each of which may be a topic within that folder.  For example, you could have a folder for family, and keep developments with each family member in a separate file.  You could have a folder for songs, and each song lyric you wrote is in that folder.

What I'm getting at is, a journal based on dates is great.  The tags make it really useful.  It would be better if we could also, in addition to these useful features, create hierarchies of topics that are not based on date.

**Comment by jendrikseipp** [comment]
Thanks for the suggestion. But the feature you describe is out of scope for a journal application like RedNotebook. Instead, I recommend to use a tool like [Logseq](https://logseq.com/).
